infinispan-client/src/test/java/io/quarkus/ts/infinispan/client/OperatorOpenShiftInfinispanCountersIT.java

```diff
@@ -360,12 +363,10 @@ private void waitForAppToBeUpAndRunning(RestService node) {
      * Tested is only the right returned status code.
      */
     private String getCounterValue(RestService node, String url) {
-        String actualResponse = node.given()
-                .get(url)
-                .then().statusCode(HttpStatus.SC_OK)
-                .extract().asString();
-
-        return actualResponse;
+        Response response = node.given().get(url);
+        String body = response.body().asString();
+        Assertions.assertEquals(HttpStatus.SC_OK, response.statusCode());
+        return body;
```

Comment:
What difference does this change make?

Reply:
Now it adds debugging in case of failure :)